lib/plugins/lint.js

```diff
@@ -0,0 +1,53 @@
+import Tooltip from "../components/tooltip";
+import { Plugin } from "prosemirror-state";
+import { Decoration, DecorationSet } from "prosemirror-view";
+
+function findProblems(schema, doc) {
+  const problems = [];
+  let lastHeadLevel = 1;
```

Comment:
I do wonder if we can improve the messaging when we refer back to H1 - is this common knowledge, why the document can only start at H2?

Reply:
I've added a special case for this instead